src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Lottery {
+    public void getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier l = new ColorSupplier();
+        ball.setColor(l.getRandomColor());
+        ball.setNumber((int)(Math.random() * 100) + 1);
```

Comment:
1. Let's use throughout our program the same methods of generating random value, e.g. `class Random` to make our program consistent and clear.

2. Please change all magic numbers with constant.

See https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery